### PR TITLE
[CDAP-5007] Feature/ui tracker entity detail

### PR DESCRIPTION
**Pull request by elfenheart**
No truncation, no copy on popup

### CODE REVIEW

**Comment by bdmogal** [comment]
With these changes, the tabs (Metadata, Lineage and Audit Log) do not stand out IMO. Its true that Metadata is underlined, but it does not give enough of a context that you're in the Metadata Tab, and can switch to Lineage or Audit Log. The other two tabs seem like mere text labels. @BradyAjay thoughts?

File: cdap-ui/app/features/tracker/tracker.less [hunk 1]
```diff
@@ -272,3 +280,246 @@ body.state-tracker {
...
+
+  .entity-container {
+    padding: 20px;
+  }
+
+  .entity-detail-tags-column, .entity-detail-schema-column, .entity-detail-properties-column {
+    position: absolute;
+    top: 178px;
+    bottom: 53px;
+    box-sizing: border-box;
```
Comment (astewes) [inline cdap-ui/app/features/tracker/tracker.less]: small but nitpicky change: use the box-sizing mixin to ensure all vendor prefixes are accounted for - .box-sizing(border-box);

File: cdap-ui/app/features/tracker/tracker.less [hunk 2]
```diff
@@ -272,3 +280,246 @@ body.state-tracker {
...
+      }
+    }
+  }
+}
+
+.tooltip.tracker-tooltip .tooltip-inner {
+  color: @title-font-color;
+  background-color: white;
+  border: none;
+  box-shadow: 0 6px 12px rgba(0,0,0,.175);
```
Comment (astewes) [inline cdap-ui/app/features/tracker/tracker.less]: mixin here as well - .box-shadow(0 6px 12px rgba(0,0,0,.175));

**Comment by astewes** [comment]
2 minor styling comments, otherwise :+1: LGTM

**Comment by elfenheart** [comment]
@astewes These will actually get automatically prefixed, since we use Autoprefixer Gulp plugin.

**Pull request merged**